<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MediaConch</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionChooseSchematron"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionConch"/>
    <addaction name="actionInfo"/>
    <addaction name="actionTrace"/>
    <addaction name="actionSchematron"/>
    <addaction name="actionPolicies"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionText"/>
    <addaction name="actionXml"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConch"/>
   <addaction name="actionInfo"/>
   <addaction name="actionTrace"/>
   <addaction name="actionSchematron"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionPolicies"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionText"/>
   <addaction name="actionXml"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionChooseSchematron">
   <property name="text">
    <string>Import Schematron</string>
   </property>
  </action>
  <action name="actionSavePolicies">
   <property name="text">
    <string>Save policies to Schematron file</string>
   </property>
  </action>
  <action name="actionConch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Conch</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionTrace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Trace</string>
   </property>
  </action>
  <action name="actionSchematron">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Apply policies</string>
   </property>
  </action>
  <action name="actionPolicies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit policies</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionXml">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
